apps/web/tailwind.config.js

```diff
@@ -47,53 +99,6 @@ module.exports = {
       montserrat: ['Montserrat', ...defaultTheme.fontFamily.sans],
       sourceCode: ['Source Code Pro', ...defaultTheme.fontFamily.mono],
     },
-    keyframes: {
-      // Dropdown menu
-      'scale-in': {
-        '0%': {opacity: 0, transform: 'scale(0)'},
-        '100%': {opacity: 1, transform: 'scale(1)'},
-      },
-      'slide-down': {
-        '0%': {opacity: 0, transform: 'translateY(-10px)'},
-        '100%': {opacity: 1, transform: 'translateY(0)'},
-      },
-      'slide-up': {
-        '0%': {opacity: 0, transform: 'translateY(10px)'},
-        '100%': {opacity: 1, transform: 'translateY(0)'},
-      },
-      // Toast
-      'toast-hide': {
-        '0%': {opacity: 1},
-        '100%': {opacity: 0},
-      },
-      'toast-slide-in-right': {
-        '0%': {transform: 'translateX(calc(100% + 1rem))'},
-        '100%': {transform: 'translateX(0)'},
-      },
-      'toast-slide-in-bottom': {
-        '0%': {transform: 'translateY(calc(100% + 1rem))'},
-        '100%': {transform: 'translateY(0)'},
-      },
-      'toast-swipe-out': {
-        '0%': {transform: 'translateX(var(--radix-toast-swipe-end-x))'},
-        '100%': {
-          transform: 'translateX(calc(100% + 1rem))',
-        },
-      },
-    },
-    animation: {
-      // Dropdown menu
-      'scale-in': 'scale-in 0.2s ease-in-out',
-      'slide-down': 'slide-down 0.6s cubic-bezier(0.16, 1, 0.3, 1)',
-      'slide-up': 'slide-up 0.6s cubic-bezier(0.16, 1, 0.3, 1)',
-      // Toast
-      'toast-hide': 'toast-hide 100ms ease-in forwards',
-      'toast-slide-in-right':
-        'toast-slide-in-right 150ms cubic-bezier(0.16, 1, 0.3, 1)',
-      'toast-slide-in-bottom':
-        'toast-slide-in-bottom 150ms cubic-bezier(0.16, 1, 0.3, 1)',
-      'toast-swipe-out': 'toast-swipe-out 100ms ease-out forwards',
-    },
```

Comment:
move inside `theme.extend` otherwise it gets rid of tailwind's default
https://tailwindcss.com/docs/animation